docs/v1/user-story

```diff
@@ -0,0 +1,124 @@
+A Year In The Life Of A Kubernetes Service Developer
+
+0. Today
+I’m building a node.js chat application for me and a couple of coworkers to goof around with.  It’s really simple.  It runs
+on one container.  To get going, I simply download a node.js container onto minikube on my laptop and started coding.  I want 
+some co-workers to be able to find the chat app to connect.  So, I register it as a service in kubernetes service.  It keeps 
+track of the IP address and the port of the app for us which is nice because I keep moving the thing around.  It started on 
+minikube on my laptop, but now it’s in the company’s kubernetes deployment on our OpenStack private cloud.  Also, I have to 
+change the port number from time to time because of conflicts, so a centralized place to find the app is really nice. 
```

Comment:
This was not an important detail.  I'll just delete the point.  I was just trying to describe what the existing service construct gave us.